ntbs-service/Services/NotificationService.cs

```diff
@@ -0,0 +1,94 @@
+using System.Threading.Tasks;

```

Comment:
I've incorporated all the patient specific stuff into the NotificationRepo/Service as we are now notification centric.